index.md

```diff
@@ -4,7 +4,7 @@ root: .
 ---
 #### What We Will Try to Do
 
-**Begin** to understand and use Git/GitHub. You will not be an expert by the end of the class. You will probably not even feel very comfortable using Git. This is okay. We want to make a start but, as with any skill, using Git takes practice.
+**Begin** to understand and use Git/GitHub. You will not be an expert by the end of the class. You will probably not even feel very comfortable using Git. This is okay. We want to make a start but, as with any skill, using Git takes practice. We will be using what we learned in The Unix Shell to use Git on the command line for this lesson.
```

Comment:
The SWC uses this language in its prerequisite section

> In this lesson we use Git from the Unix Shell. Some previous experience with the shell is expected, but isn’t mandatory.

maybe adapt that?

Reply:
I really like that sentence. 

That is exactly how I have taught these lessons in a workshop. We emphasize the relationship between the Unix Shell lesson and the Git lesson, and we use it as an opportunity to reflect back to the learners that they have made progress in accumulating new skills already.